### PR TITLE
Usability tt

### PR DESCRIPTION
**Pull request by tylertomlinson**
All Usability stories working with test 
Extension story 38 done

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -22,14 +22,15 @@ def get_review_info
...
 
   def pet_params
     params[:status] = "adoptable"
     params.permit(:image, :name, :description, :age, :sex, :status)
   end
 
   def review_params
+    params.delete(:image) if params[:image].empty?
     params.permit(:title, :rating, :content, :image)
   end
```
Comment (StephanieFriend) [inline app/controllers/application_controller.rb:35]: Looks nice!

File: db/migrate/20200331030355_add_default_image_to_reviews.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddDefaultImageToReviews < ActiveRecord::Migration[5.1]
+  def change
+    change_column :reviews, :image, :string, default: "https://www.eaglenewsonline.com/wp-content/uploads/2018/12/paws-tongue.jpg"
+  end
```
Comment (StephanieFriend) [inline db/migrate/20200331030355_add_default_image_to_reviews.rb:4]: Love it!

File: spec/features/pets/user_can_visit_pet_show_page_by_clicking_pet_name_spec.rb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    it "favorites page takes me to that pet's show page" do
+
+      visit "/pets/#{@pet_1.id}"
+      click_on "Favorite This Pet"
+
+      visit "/favorites"
+
+      click_link("#{@pet_1.name}")
+
+      expect(current_path).to eq("/pets/#{@pet_1.id}")
```
Comment (StephanieFriend) [inline spec/features/pets/user_can_visit_pet_show_page_by_clicking_pet_name_spec.rb:54]: Tests look great!

File: spec/features/reviews/default_picture_spec.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+#   visit "/shelters/#{@shelter_1.id}/reviews"
+#
+#   fill_in 'Title', with: "Test"
+#   fill_in 'Rating', with: 5
+#   fill_in 'Content', with: "Test"
+#
+#   click_on 'Submit'
+#
+#   expect(page).to have_xpath("//img[contains(@src,'https://www.eaglenewsonline.com/wp-content/uploads/2018/12/paws-tongue.jpg')]")
+# end
```
Comment (StephanieFriend) [inline spec/features/reviews/default_picture_spec.rb:15]: Let us remember to come back to this one

**Review by StephanieFriend — APPROVED** [review]
Everything looks great! Now lets focus on the testing. :D

File: db/migrate/20200331030355_add_default_image_to_reviews.rb (hunk 2 above)
Comment (tylertomlinson) [reply db/migrate/20200331030355_add_default_image_to_reviews.rb:4]: Yes. Super easy to do

File: spec/features/reviews/default_picture_spec.rb (hunk 4 above)
Comment (tylertomlinson) [reply spec/features/reviews/default_picture_spec.rb:15]: I have notes on which tests need to be working. Added and cried

**Pull request merged**